### PR TITLE
Include departments with no vacancies

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Fixed missing departments
- Fixed 404ing career-explorer
- Removed links to working here pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-779.demos.haus/careers
- See that Administration and Legal show up in overview and take you to the correct dept page
- View the site locally in your web browser at: https://canonical-com-779.demos.haus/careers/career-explorer
- See that the page renders successfully 

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-1734

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -210,7 +217,7 @@ def job_details(job_id, job_title):
     return flask.render_template("/careers/job-detail.html", **context)
 
 
-@app.route("/careers/start")
+@app.route("/careers/career-explorer")
```
Comment (anthonydillon) [inline webapp/app.py:220]: There are links to the old page on the homepage and navigation which leads to a 404. Can you add a redirect and update links across this project.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
When clicking through the button to "See all Administration roles" does not show no roles.

**Comment by mtruj013** [comment]
Thanks @anthonydillon! I've addressed your comments if you'd like to take another look?

**Review by anthonydillon — APPROVED** [review]
LGTM thanks @mtruj013 👍

**Comment by marinacastejon** [comment]
LGTM, thanks @mtruj013

**Comment by mtruj013** [comment]
Regarding the changed CTAs: It decided via MM that they should be changed to "Show all roles" as these departments do not currently have open roles

**Pull request merged**